process/smartContract/process.go

```diff
@@ -49,6 +49,7 @@ const (
 	generalSCRIdentifier = "writeLog"
 	signalError          = "signalError"
 	completedTxEvent     = "completedTxEvent"
+	returnOkData         = "@6f"
```

Comment:
this is misleading. It is not the okData but a prefix. The ok return data is `@6f6b`

Reply:
done